<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventForm</class>
 <widget class="QWidget" name="inventForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>631</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="inventLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
